src/context/auth/authStorage/AuthStorage.ts

```diff
@@ -1,82 +1,93 @@
-import { Storage } from './AuthStorage.types';
+import { Storage, TokenData } from './AuthStorage.types';
 
 const ACCESS_TOKEN_KEY = 'accessToken';
 const REFRESH_TOKEN_KEY = 'refreshToken';
 const EXPIRES_KEY = 'expires';
 
+const defaultTokenData = {
+  accessToken: null,
+  refreshToken: null,
+  expires: null,
+} satisfies TokenData;
 class AuthStorage {
-  private _accessToken: string | null = null;
-  private _refreshToken: string | null = null;
-  private _expires: number | null = null;
-  private _storage: Storage | null = null;
+  private _storage: Storage | null;
```

Comment:
Explicit typing can be removed, a type will be infered based on the assignment in the constructor
```suggestion
  private _storage;
```